client/homebrew/brewRenderer/brewRenderer.less

```diff
@@ -31,6 +34,12 @@
 
 .pane { position : relative; }
 
+.pane.previewPane {
+	display: grid;
+	grid-template-columns: 1fr;
+	grid-template-rows: 32px 1fr;

```

Comment:
can this just be 'auto' or something, for the height, so that it just accommodates whatever height the toolbar is (as set by toolbar.less)?  Rather than setting it here at a magic number like 32px?

Reply:
grid is not my forte, but it should be possible.

I must say, our structure is set to work like this in other places, like the navbar or the snippetbar, both heights are hardcoded everywhere, instead of allowing for any flow.